frontend/app/routes/protected/apply/$id/child/children/$childId/cannot-apply-child.tsx

```diff
@@ -0,0 +1,89 @@
+import { redirect, useFetcher } from 'react-router';
+
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { Trans, useTranslation } from 'react-i18next';
+
+import type { Route } from './+types/cannot-apply-child';
+
+import { TYPES } from '~/.server/constants';
+import { loadProtectedApplySingleChildState } from '~/.server/routes/helpers/protected-apply-child-route-helpers';
+import { getFixedT } from '~/.server/utils/locale.utils';
+import { ButtonLink } from '~/components/buttons';
+import { CsrfTokenInput } from '~/components/csrf-token-input';
+import { LoadingButton } from '~/components/loading-button';
+import { pageIds } from '~/page-ids';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { mergeMeta } from '~/utils/meta-utils';
+import type { RouteHandleData } from '~/utils/route-utils';
+import { getPathById } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('protected-apply-child', 'gcweb'),
+  pageIdentifier: pageIds.protected.apply.child.cannotApplyChild,
+  pageTitleI18nKey: 'protected-apply-child:children.cannot-apply-child.page-title',
+} as const satisfies RouteHandleData;
+
+export const meta: Route.MetaFunction = mergeMeta(({ data }) => {
+  return getTitleMetaTags(data.meta.title);
+});
+
+export async function loader({ context: { appContainer, session }, params, request }: Route.LoaderArgs) {
```

Comment:
All pages in the protected route should call `validateAuthSession` in both loader and action.

Reply:
Oh I didn't know! Ok sounds good! Adding them now